src/NewCommand.php

```diff
@@ -28,7 +28,8 @@ protected function configure()
             ->setDescription('Create a new Laravel application')
             ->addArgument('name', InputArgument::OPTIONAL)
             ->addOption('dev', null, InputOption::VALUE_NONE, 'Installs the latest "development" release')
-            ->addOption('force', 'f', InputOption::VALUE_NONE, 'Forces install even if the directory already exists');
+            ->addOption('force', 'f', InputOption::VALUE_NONE, 'Forces install even if the directory already exists')
+            ->addOption('preset', null, InputOption::VALUE_REQUIRED, 'Choose a preset list of packages to install along with your new Laravel installtion');
```

Comment:
I don't think it should be required, as we may still want to install plain Laravel.

Reply:
You can still run `laravel new`, but providing the option without a value is not allowed:

okay: `laravel new --preset=media`
not okay: `laravel new --preset`